Microsoft.Toolkit.Uwp.UI.Controls/UniformGrid/UniformGrid.Properties.cs

```diff
@@ -0,0 +1,236 @@
+        }
+
+        /// <summary>
+        /// Sets the AutoLayout Attached Property Value. Used internally to track items which need to be arranged vs. fixed in place.
+        /// </summary>
+        /// <param name="element"><see cref="RowDefinition"/></param>
+        /// <param name="value">A true value indicates this item should be automatically arranged.</param>
+        public static void SetAutoLayout(RowDefinition element, bool? value)
+        {
+            element.SetValue(AutoLayoutProperty, value);
+        }
+
+        /// <summary>
+        /// Gets the AutoLayout Attached Property Value. Used internally to track items which need to be arranged vs. fixed in place.
+        /// </summary>
+        /// <param name="element"><see cref="RowDefinition"/></param>
+        /// <returns>A true value indicates this item should be automatically arranged.</returns>
+        public static bool? GetAutoLayout(RowDefinition element)
+        {
+            return (bool?)element.GetValue(AutoLayoutProperty);
+        }
+
+        //// This Property is for ColumnDefinition (DependencyObject) vs. the Grid's FrameworkElement attached property.
+        #pragma warning disable CS0108 // Member hides inherited member; missing new keyword
+        /// <summary>
+        /// Determines if this element in the grid participates in the auto-layout algorithm.
+        /// </summary>
+        public static readonly DependencyProperty ColumnProperty =
+        #pragma warning restore CS0108 // Member hides inherited member; missing new keyword
+            DependencyProperty.RegisterAttached(
+              "Column",
+              typeof(int),
+              typeof(UniformGrid),
+              new PropertyMetadata(null));
+
+        /// <summary>
+        /// Sets the Column Attached Property Value.
+        /// </summary>
+        /// <param name="element"><see cref="ColumnDefinition"/></param>
+        /// <param name="value">Zero-based Index of column for this ColumnDefinition.</param>
+        public static void SetColumn(ColumnDefinition element, int value)
+        {
+            element.SetValue(ColumnProperty, value);
+        }
+
+        /// <summary>
+        /// Gets the Column Attached Property Value.
+        /// </summary>
+        /// <param name="element"><see cref="ColumnDefinition"/></param>
+        /// <returns>Specified column index for the ColumnDefinition.</returns>
+        public static int GetColumn(ColumnDefinition element)
+        {
+            return (int)element.GetValue(ColumnProperty);
+        }
+
+        //// This Property is for RowDefinition (DependencyObject) vs. the Grid's FrameworkElement attached property.
+        #pragma warning disable CS0108 // Member hides inherited member; missing new keyword
+        /// <summary>
+        /// Determines if this element in the grid participates in the auto-layout algorithm.
+        /// </summary>
+        public static readonly DependencyProperty RowProperty =
```

Comment:
Why the need to hide the existing Row property?

Reply:
Same as Column.